app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb

```diff
@@ -97,21 +97,50 @@ def convert_to_group_trigger(operation, miq_alert)
                                       })
     end
 
+    MW_WEB_SESSIONS = %w(
+      mw_aggregated_active_web_sessions
+      mw_aggregated_expired_web_sessions
+      mw_aggregated_rejected_web_sessions
+    ).freeze
+
+    MW_DATASTORE = %w(
```

Comment:
Ah, I am seeing this now.
Datastore sounds odd in Middleware terminology, I think is better to change this to Datasource.

Reply:
I see several places used this term, here and in tests, only comment here to not create noise.

Reply:
I think It was a mistake by me side